Source/UnitTests/UnitTests.vcxproj

```diff
@@ -59,16 +59,51 @@
       <SubSystem>Console</SubSystem>
     </Link>
   </ItemDefinitionGroup>
+  <ItemGroup>
+    <ClInclude Include="Core\DSP\DSPTestBinary.h" />
+    <ClInclude Include="Core\DSP\DSPTestText.h" />
+    <ClInclude Include="Core\DSP\HermesBinary.h" />
+    <ClInclude Include="Core\IOS\ES\TestBinaryData.h" />
+  </ItemGroup>
   <ItemGroup>
     <!--gtest is rather small, so just include it into the build here-->
     <ClCompile Include="$(ExternalsDir)gtest\src\gtest-all.cc" />
     <ClCompile Include="$(ExternalsDir)gtest\src\gtest_main.cc" />
     <!--Lump all of the tests (and supporting code) into one binary-->
-    <ClCompile Include="*.cpp" />
-    <ClCompile Include="*\*.cpp">
-      <ExcludedFromBuild Condition="'$(Platform)'!='x64'">true</ExcludedFromBuild>
-    </ClCompile>
-    <ClCompile Include="*\*\*.cpp" />
+    <ClCompile Include="Common\BitFieldTest.cpp" />
+    <ClCompile Include="Common\BitSetTest.cpp" />
+    <ClCompile Include="Common\BitUtilsTest.cpp" />
+    <ClCompile Include="Common\BlockingLoopTest.cpp" />
+    <ClCompile Include="Common\BusyLoopTest.cpp" />
+    <ClCompile Include="Common\CommonFuncsTest.cpp" />
+    <ClCompile Include="Common\Crypto\EcTest.cpp" />
+    <ClCompile Include="Common\EventTest.cpp" />
+    <ClCompile Include="Common\FixedSizeQueueTest.cpp" />
+    <ClCompile Include="Common\FlagTest.cpp" />
+    <ClCompile Include="Common\FloatUtilsTest.cpp" />
+    <ClCompile Include="Common\MathUtilTest.cpp" />
+    <ClCompile Include="Common\NandPathsTest.cpp" />
+    <ClCompile Include="Common\SPSCQueueTest.cpp" />
+    <ClCompile Include="Common\StringUtilTest.cpp" />
+    <ClCompile Include="Common\SwapTest.cpp" />
```

Comment:
Extra line?

Reply:
what?

Reply:
i think you're indicating i should insert a blank line to delineate the difference between root dirs here? I would prefer not to.